templates/desktop/statistics.html

```diff
@@ -32,214 +29,198 @@ <h1>Ubuntu user statistics</h1>
+      <h2 class="p-heading--four">Display server</h2>
+      <p>The display server controls the input and output of its clients to and from the rest of the operating system and the hardware.</p>
+      <div class="u-align--center">
+        <svg id="display-server"></svg>
+      </div>
     </div>
   </div>
 </section>
-<section class="p-strip is-shallow is-bordered">
+<section class="p-strip--light is-bordered">
+  <div class="row u-equal-height u-vertically-center">
+    <div class="col-6">
+      <img src="https://via.placeholder.com/350x150" alt="placeholder image for chart"/>
+    </div>
+    <div class="col-6">
+      <h3 class="p-heading--four">Firmware</h3>
+      <p>The display server controls the input and output of its clients to and from the rest of the operating system and the hardware.</p>
+    </div>
+  </div>
+</section>
+<section id="user-report" class="p-strip is-shallow is-bordered">
   <div class="row">
-    <h3>What graphics setup do users have?</h2>
+    <div class="col-6">
+      <h3>What graphics setup do users have?</h3>
+      <p>*Data for graphics setup only applies to physical machines</p>
+    </div>
   </div>
-  <div class="row u-equal-height">
-    <div class="col-6 prefix-2">
-      <div class="p-card--flex">
-        <div id="one-screen" class="p-card__center-content"></div>
-        <p class="u-no-margin--bottom">This was detected on install so many users may have connected screens after the installing Ubuntu.</p>
-      </div>
+  <div class="row u-equal-height u-vertically-center p-divider">
+    <div class="col-6 p-divider__block">
+      <img src="https://via.placeholder.com/350x150" alt="placeholder image for chart"/>
+      <p>*On install. This was detected on install so many users may have connected screens after the installing Ubuntu.</p>
     </div>
-    <div class="col-6 suffix-2">
-      <div class="p-card--flex">
-        <div id="one-gpu" class="p-card__center-content"></div>
-        <p class="u-no-margin--bottom">Graphical Processing Unit (GPU) is used to generate videos and images for displaying on screen.</p>
-      </div>
+    <div class="col-6 p-divider__block">
+      <img src="https://via.placeholder.com/350x150" alt="placeholder image for chart"/>
+      <p>Graphical Process Unit (GPU) is used to generate videos and images for displaying on screen</p>
     </div>
   </div>
 </section>
-<section class="p-strip--light">
+<section class="p-strip--light is-bordered">
   <div class="row">
-    <h3>Popular screen sizes</h2>
+    <h3>Popular screen sizes</h3>
     <p>This data set shows the 15 most popular screen sizes.</p>
   </div>
   <div class="row u-equal-height">
-    <svg id="popular-screen-sizes" height="320"></svg>
-  </div>
-</section>
-<section class="p-strip--light is-bordered u-no-padding--top">
-  <div class="row u-equal-height">
+    <div class="col-5">
+      <svg id="popular-screen-sizes" height="320"></svg>
+    </div>
+    <div class="col-1 suffix-1"></div>
```

Comment:
suffix adds extra padding inside the current column therefore this should be added to the column above and change that from `col-5` to `col-6`